react/react1/week1/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from 'react';
+import ToDoList from './ToDoList'
+
+function App() {
+  const [todos, setTodos] = useState([
+    {
+      id: 1,
+      description: "Get out of bed",
+      deadline: 'Wed Sep 13 2017'
+      
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+      deadline: 'Thu Sep 14 2017'
+      
+
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      deadline: 'Fri Sep 15 2017'
+      
+
+    },
+  ])
```

Comment:
You can create separate component for the todos and export, so that you can use here as props also, but you can also do what you have done :)